src/Factory/AwsFactory.php

```diff
@@ -16,18 +15,19 @@ class AwsFactory implements FactoryInterface
 {
     /**
      * @param ContainerInterface $container
-     * @param string             $requestedName
-     * @param array|null         $options
+     * @param string $requestedName
+     * @param array|null $options
      * @return AwsSdk
      */
     public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
         // Instantiate the AWS SDK for PHP
         $config = $container->get('Config');
         $config = isset($config['aws']) ? $config['aws'] : [];
+
         $config += [
             'ua_append' => [
-                'ZF2/' . Version::VERSION,
+                'ZF2/' . Module::VERSION,
```

Comment:
Oh I see, thank you all for the explanation here! Then it makes sense. Alright, now it looks good to me, I'll merge it, add documentation in `README.md` and tag a release for this. :)